tools/wptrunner/wptrunner/config.py

```diff
@@ -1,11 +1,12 @@
-from configparser import SafeConfigParser
+from configparser import ConfigParser
 import os
 import sys
 from collections import OrderedDict
+from typing import Any, Dict
 
 here = os.path.dirname(__file__)
 
-class ConfigDict(dict):
+class ConfigDict(Dict[str, Any]):
```

Comment:
This feels weird, but I guess it's right.

Reply:
Do you mean it's weird to use a type as the base class? I agree, things don't work quite as I first guessed with inheritance. It's not possible to do `dict: Dict[str, Any]`, that's a syntax error. My guess is since that aside from ABCs and maybe generics (if Python has those) inheritance is from a specific type, not a type in the mypy sense.

And `typing.Dict` can be used like this, by some magic I haven't inspected closely.